### PR TITLE
add Taint and Toleration

### PR DESCRIPTION
**Pull request by haoqing0110**
Ref: https://github.com/open-cluster-management-io/community/issues/48

Signed-off-by: haoqing0110 <qhao@redhat.com>

### CODE REVIEW

File: content/en/concepts/managedcluster.md [hunk 1]
```diff
@@ -123,6 +123,81 @@ in the status of the `ManagedCluster`, e.g.:
     conditions: ...
 ```
 
+### Cluster taints and tolerations
+
+To support filtering unhealthy/not-reporting clusters and keep workloads from
+being placed in unhealthy or unreachable clusters, we introduce the similar
```
Comment (qiujian16) [inline content/en/concepts/managedcluster.md:129]: It also allows user to add customized a taint to deselect a cluster from placement. This is useful when user wants to set a cluster to maintanence mode and evict workload from this cluster.

Comment (haoqing0110) [reply content/en/concepts/managedcluster.md:129]: Updated.

File: content/en/concepts/managedcluster.md [hunk 2]
```diff
@@ -123,6 +123,83 @@ in the status of the `ManagedCluster`, e.g.:
...
+concept of taint/toleration in Kubernetes. It also allows user to add customized
+a taint to deselect a cluster from placement. This is useful when user wants to
```
Comment (qiujian16) [inline content/en/concepts/managedcluster.md:131]: a customized taint

This is useful when the user ...

Comment (haoqing0110) [reply content/en/concepts/managedcluster.md]: fixed.

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**